lnwallet/channel.go

```diff
@@ -2435,7 +2443,9 @@ func (lc *LightningChannel) evaluateHTLCView(view *htlcView, ourBalance,
 	theirBalance *lnwire.MilliSatoshi, nextHeight uint64,
 	remoteChain, mutateState bool) *htlcView {
 
-	newView := &htlcView{}
+	newView := &htlcView{
+		feePerKw: view.feePerKw,
```

Comment:
How can we know what the fee rate is before we've fully evaluated the view?

Reply:
This is the initial fee rate. If `fee_updates` are found among the log updates, it will be updated, otherwise it will stay the same. 

Added a comment.